### PR TITLE
Receivebles page

### PR DESCRIPTION
**Pull request by konstantin1390**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

File: src/client/rewards/Payables/Payables.js [hunk 1]
```diff
@@ -4,14 +4,13 @@ import { injectIntl } from 'react-intl';
...
-import UserPayableCard from './UserPayableCard/UserPayableCard';
+import PayableCard from './PayableCard/PayableCard';
 import './Payables.less';
 
 const Payables = ({ intl, userName, currentSteemDollarPrice, filterData }) => {
   const payableFilters = {};
   _.map(filterData, f => {
-    payableFilters.days = f.filterName === 'days' ? f.value : '';
-    payableFilters.payable = f.filterName === 'payable' ? f.value : '';
+    payableFilters[`${f.filterName}`] = f.value;
```
Comment (milklim) [inline src/client/rewards/Payables/Payables.js]: unnecessary string interpolation

File: src/waivioApi/ApiClient.js [hunk 2]
```diff
@@ -603,12 +603,13 @@ export const getAuthenticatedUserMetadata = (
 };
 
 export const getLenders = ({ sponsor, user, filters }) => {
-  const payable = filters && filters.payable ? `&payable=${filters.payable}` : ``;
-  const days = filters && filters.days ? `&days=${filters.days}` : ``;
-  const isUser = !user ? `?sponsor=${sponsor}` : `?sponsor=${sponsor}&userName=${user}`;
+  const isSponsor = sponsor ? `?sponsor=${sponsor}` : '';
+  const payable = filters && filters.payable ? `&payable=${filters.payable}` : '';
+  const days = filters && filters.days ? `&days=${filters.days}` : '';
+  const isUser = user ? (sponsor ? `&userName=${user}` : `?userName=${user}`) : '';
```
Comment (milklim) [inline src/waivioApi/ApiClient.js]: check this condition

File: src/client/rewards/Rewards.js [hunk 3]
```diff
@@ -411,22 +411,34 @@ class Rewards extends React.Component {
...
+              <div className="right">
+                {!isEmpty(this.props.userLocation) && !isCreate && (
+                  <React.Fragment>
+                    <MapWrap
+                      wobjects={this.getRequiredObjects()}
+                      userLocation={this.props.userLocation}
+                      onMarkerClick={this.goToCampaign}
+                      getAreaSearchData={this.getAreaSearchData}
+                    />
+                  </React.Fragment>
```
Comment (milklim) [inline src/client/rewards/Rewards.js]: remove Fragment wrapper

Comment (konstantin1390) [reply src/client/rewards/Rewards.js]: done

File: src/client/rewards/Payables/Payables.js (hunk 1 above)
Comment (konstantin1390) [reply src/client/rewards/Payables/Payables.js]: done

**Pull request merged**